app/services/organizations/update_service.rb

```diff
@@ -10,6 +10,13 @@ def initialize(organization:, params:)
     end
 
     def call
+      if params.key?(:document_number_prefix) && !params[:document_number_prefix].to_s.length.between?(1, 10)
```

Comment:
Is there a reason for not adding this as a validation on the organization model?

Reply:
yep, This field is initially set with `after_create` hook and in order to set this field correctly to default value that we currently use, we need record's ID (that's why we are using after create). If we add it on model level, this validation will raise error with each creation since validation will be called even before after_create hook is triggered...

Reply:
I see.

For update only validation, it's possible to use the `on` option on the validates instruction:

```ruby
validates ..., on: :update`
```

Reply:
Cool, thanks, forgot on `on `option